src/components/RangeInput.vue

```diff
@@ -11,7 +11,7 @@
     >
       <form
         :class="suit('form')"
-        @submit.prevent="refine({ min: minInput, max: maxInput })"
+        @submit.prevent="refine({ min: minInput || values.min, max: maxInput || values.max })"
```

Comment:
I believe that this doesn't fully fix the bug. I think that it'll fix everything when a refresh is done, but the input will still behave strangely when you hit the browser's back button.

Let's say that you a range input:

1.  You put in min: 10, max: 100
1.  You hit Go
1.  You change min to 20
1.  You hit Go again
1.  You hit the back button on the browser (with the history router the page will not reload but the state of the search changes)
1.  You hit Go again

In this scenario, the min value will be reset to 60.

I believe that this will still happen with your fix. In that scenario `minValue` would have been set to `60` and `values.min` would have been set to `50`.

Reply:
What if you added a watcher on `value.min` & `value.max`? In that watcher you could update `minValue` & `maxValue` every time `value.min` & `value.max` get updated by any source.

Reply:
> In this scenario, the min value will be reset to 60.
> I believe that this will still happen with your fix. In that scenario minValue would have been set to 60 and values.min would have been set to 50.

Do you mean it will be reset to 20, while it should've stayed as 10, right?
You're right. I didn't think of that path. Thanks!